fml/thread.h

```diff
@@ -16,7 +17,52 @@ namespace fml {
 
 class Thread {
  public:
-  explicit Thread(const std::string& name = "");
+  // Valid values for priority of Thread
```

Comment:
All comments should be complete sentences with punctuation. https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar